### ISSUE TITLE
Baseline results

### ISSUE THREAD

**Issue by YovaKem** [babylm/evaluation-pipeline#16]
The approaches we tried for the shared task didn't yield an improvement over our baselines so we didn't make a submission at all, but I thought I'd share our baseline results with RoBERTa-base for reference, as they outperform the ones reported by the task organizers on many evaluation tasks, especially from BLiMP. 

**Strict-small Track**

*BLiMP*
| Model | Anaphor Agr. | Agr. Structure | Binding | Control/Raising | D-N Agr. | Ellipsis | Filler-Gap | Irregular Forms | Island Effects | NPI Licensing | Quantifiers | S-V Agr. |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| RoBERTa-base | 87.2 | 72.9 | 69.8 | 69.1 | 95 | 74.3 | 74.3 | 86.5 | 46.0 | 54.9 | 75.3 | 75.8|

*(Super)GLUE*
| Model | CoLA | SST-2 | MRPC (F1) | QQP (F1) | MNLI | MNLI-mm | QNLI | RTE | BoolQ | MultiRC | WSC |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| RoBERTa-base | 73.9 | 87.4 | 81.3 | 78.5 | 68.2 |  68.4 | 61.6 | 48.5 | 65.0 | 60.0 | 61.5|

The model has a RoBERTa-base architecture (with pre-layer normalization); it is initialized from scratch, and trained with the following hyperparameters:

```
  --max_seq_length 128 
  --batch_size 4096
  --optim adamw_torch 
  --lr_scheduler_type linear 
  --learning_rate 4e-4 
  --warmup_ratio 0.01 
  --weight_decay 0.01 
  --adam_beta1 0.9 
  --adam_beta2 0.98 
  --adam_epsilon 1e-8 
  --max_grad_norm 1.0 
  --mlm_probability 0.15
```

The tokenizer is trained from scratch on the 10M corpus, setting the vocab size to the 50304.
The model is trained on the 10M babylm corpus (excluding lines with less than 10 characters, empty lines and duplicates). 

NB: a higher `max_seq_length` might yield better results, since 128 is a bit restrictive wrt some of the evaluation tasks, esp. in GLUE.
